public/pages/DetectorConfig/containers/MetaData.tsx

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import ContentPanel from '../../../components/ContentPanel/ContentPanel';
+import {
+  //@ts-ignore
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiText,
+  EuiFormRow,
+  EuiLink,
+  EuiButton,
+  EuiFormRowProps,
+} from '@elastic/eui';
+import { PLUGIN_NAME } from '../../../utils/constants';
+import {
+  Detector,
+  Schedule,
+  UiMetaData,
+  FILTER_TYPES,
+  UIFilter,
+} from '../../../models/interfaces';
+import React, { Component, FunctionComponent } from 'react';
+import { displayText } from '../../createDetector/components/DataFilters/utils/helpers';
+import { CodeModal } from '../components/CodeModal/CodeModal';
+
+interface MetaDataProps {
+  detectorId: string;
+  detector: Detector;
+  onEditDetector(): void;
+}
+
+const FixedWidthRow = (props: EuiFormRowProps) => (
+  <EuiFormRow {...props} style={{ width: '250px' }} />
+);
+
+interface ConfigCellProps {
+  title: string;
+  description: string | string[];
+}
+
+const ConfigCell: FunctionComponent<ConfigCellProps> = (
+  props: ConfigCellProps
+) => {
+  return (
+    <FixedWidthRow label={props.title}>
+      <EuiText>
+        <p className="enabled">{props.description}</p>
+      </EuiText>
+    </FixedWidthRow>
+  );
+};
+
+function toString(obj: { period: Schedule }): string;
+function toString(utcEpochSeconds: number): string;
+function toString(filter: UiMetaData): string;
```

Comment:
what's this for? seems to be overridden by `toString(obj: any)` below

Reply:
not sure now.  removed.